### ISSUE TITLE
Add load more button on org page to be similar to home page

### ISSUE THREAD

**Issue by ecurrencyhodler** [stakwork/sphinx-tribes#1286]
### Context
Add load more button on org page to be similar to home page. Here is an example org: https://people-test.sphinx.chat/org/bounties/clf6qmo4nncmf23du7ng

Pagination is now available for orgs. The call is: https://people-test.sphinx.chat/organizations/bounties/{org-uuid}

### Design
This except all orgs displayed should be the same.
<img width="1185" alt="Screenshot 2024-01-08 at 10 15 17 AM" src="https://github.com/stakwork/sphinx-tribes/assets/32662508/ca991910-f48b-487c-80b2-287076d8a127">

### Acceptance Criteria
- [ ] I've tested on Chrome
- [ ] I've submitted a screenshot or recording in my pr
- [ ] I've created a component test that the "load more" is visible
- [ ] When I click on the button, it makes a request

Here is an [example unit test](https://github.com/stakwork/sphinx-tribes/blob/9310f49b3b17a51992dada932f4298eb9eba15ff/frontend/app/src/people/widgetViews/__tests__/AboutView.spec.tsx)

**Comment by hritikmunde** [comment]
Can I take this?

**Comment by ecurrencyhodler** [comment]
This is currently blocked.

**Comment by hritikmunde** [comment]
Okay can you assign it to me after it is unblocked?

**Comment by ecurrencyhodler** [comment]
Closing in favor of https://github.com/stakwork/sphinx-tribes-frontend/issues/42
